src/notes/FluxNotesEditor.scss

```diff
@@ -24,12 +26,17 @@
         p.note-name {
             text-align: left;
             padding-top:0;
-            font-size: .8rem;
-            margin: 0px 0px 0px 10px;
+            font-size: 0.8rem;
+            margin: 0px;
+            padding-left: 0.5rem;
             font-weight: bold;
             
-            #edit-note-name-btn:hover {
-                cursor: pointer;
+            #edit-note-name-btn {
+                margin-left: -23px;
+                margin-right: 3px;
```

Comment:
[MG1] Very small thing, but I would consider playing around with the margin-right value a little more. I think the text looks a bit more aligned with "Signed By" and "Source" with a value of 7px or 6px. Right now the note name is slightly to the left of the rest of the text.